src/components/ui/ControlBar/ControlBar.mdx

```diff
@@ -15,7 +15,7 @@ You can place ControlBar into the parent element according to the `layout` prop,
 
 The ControlBar component contains one or more `ControlBarItem` components, which can show a label and icon. You can add a dropdown list to each ControlBarItem by setting the [popOver props](/?path=/docs/ui-components-popover--basic-pop-over-menu).
```

Comment:
Should we update ControlBarItem reference in this doc to ControlBarButton?

Reply:
It'll make this PR much busier, but sure

Reply:
Sorry, i just meant to update it in documentation (the mdx file and not everything) :)